src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/.github/workflows/build-release.yml

```diff
@@ -13,10 +13,13 @@ on:
   workflow_dispatch:
 
 env:
-  MAIN_PYTHON_VERSION: "3.8"
+  MAIN_PYTHON_VERSION: "3.9"
   DOCUMENTATION_CNAME: ""
   POETRY_HTTP_BASIC_SOLUTIONS_PRIVATE_PYPI_USERNAME: "PAT"
```

Comment:
Is this still called `SOLUTIONS` or did we rename to `PYANSYS` ?

Reply:
Good catch! Fixed.